web/frontend/src/App.svelte

```diff
@@ -6,19 +6,40 @@
   import { fade } from "svelte/transition";
   export let menu = 1;
 
+  let current_season = 3
+  let newest_episode = 8
+  let participants = []
   let matches = []
 
   onMount(async () => {
-  fetch("http://dayo-project.herokuapp.com/api/v1/matches")
-  .then(response => response.json())
-  .then(data => {
-    matches = data["data"][0]["matches"];
-    console.log(matches)
-  }).catch(error => {
-    console.log(error);
-    return 0;
+    fetch("http://dayo-project.herokuapp.com/api/v1/participants")
+    .then(response => response.json())
+    .then(data => {
+      participants = data["data"];
+      participants = participants.filter(participant => {
+        return participant.season === current_season;
+      })
+    }).catch(error => {
+      console.log(error);
```

Comment:
Don't return error. Use custom error instead. Error will leak database details

Reply:
We might have to check all files. I am sure we committed a few of those before but we should change them imo